<ui version="4.0" >
 <class>CreateGameDialog</class>
 <widget class="QDialog" name="CreateGameDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>407</width>
    <height>187</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Create Game</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QGridLayout" name="gridLayout" >
     <item row="0" column="0" >
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Name:</string>
       </property>
       <property name="buddy" >
        <cstring>lineEditGameName</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="lineEditGameName" />
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Description:</string>
       </property>
       <property name="buddy" >
        <cstring>lineEditGameDescription</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLineEdit" name="lineEditGameDescription" />
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Number of players:</string>
       </property>
       <property name="buddy" >
        <cstring>spinBoxMinPlayers</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <layout class="QHBoxLayout" name="horizontalLayout" >
       <item>
        <widget class="QSpinBox" name="spinBoxMinPlayers" />
       </item>
       <item>
        <widget class="QLabel" name="label_4" >
         <property name="maximumSize" >
          <size>
           <width>16</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text" >
          <string>-</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBoxMaxPlayers" />
       </item>
      </layout>
     </item>
     <item row="3" column="0" >
      <widget class="QLabel" name="label_5" >
       <property name="text" >
        <string>Passport:</string>
       </property>
       <property name="buddy" >
        <cstring>checkBoxEnablePlayerPassword</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1" >
      <layout class="QHBoxLayout" name="horizontalLayout_2" >
       <item>
        <widget class="QCheckBox" name="checkBoxEnablePlayerPassword" >
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditPlayerPassword" />
       </item>
      </layout>
     </item>
     <item row="4" column="0" >
      <widget class="QLabel" name="label_6" >
       <property name="text" >
        <string>Shuffle players:</string>
       </property>
       <property name="buddy" >
        <cstring>checkBoxShufflePlayers</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="1" >
      <widget class="QCheckBox" name="checkBoxShufflePlayers" >
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" >
     <item>
      <widget class="QLabel" name="label_7" >
       <property name="text" >
        <string>Player name:</string>
       </property>
       <property name="buddy" >
        <cstring>lineEditPlayerName</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditPlayerName" />
     </item>
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCreate" >
       <property name="text" >
        <string>Create</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCancel" >
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditGameName</tabstop>
  <tabstop>lineEditGameDescription</tabstop>
  <tabstop>spinBoxMinPlayers</tabstop>
  <tabstop>spinBoxMaxPlayers</tabstop>
  <tabstop>checkBoxEnablePlayerPassword</tabstop>
  <tabstop>lineEditPlayerPassword</tabstop>
  <tabstop>checkBoxShufflePlayers</tabstop>
  <tabstop>lineEditPlayerName</tabstop>
  <tabstop>pushButtonCreate</tabstop>
  <tabstop>pushButtonCancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonCancel</sender>
   <signal>clicked()</signal>
   <receiver>CreateGameDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>345</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel" >
     <x>311</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
